### DISCUSSION TITLE
USB Cul Stick wird nicht erkannt (Raspberrymatic Home Assistant Add-On)

### DISCUSSION THREAD

**Discussion by slupus** [OpenCCU/OpenCCU#2647] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Der USB Cul Stick wird von Home Assistant erkannt und als ttyACM1 angezeigt.
Das Raspberrymatic Addon erkennt leider keinen Stick (Identifying Homematic RF-Hardware: HmRF: n/a, HmIP: n/a, OK).
Aufgrund von [#1745](https://github.com/jens-maus/RaspberryMatic/issues/1745) bin ich davon ausgegangen, dass die Kombination funktioniert.

### Describe the behavior you expected

Raspberrymatic erkennt den Stick, sodass er als Zentrale angelernt werden kann.

### Steps to reproduce the issue

1. Raspberrymatic Addon im Homeassistant starten
2. Protokoll des Addons auslesen

### What is the version this bug report is based on?

3.73.9.20240130

### Which base platform are you running?

ha-addon (HomeAssistant Add-on)

### Which HomeMatic/homematicIP radio module are you using?

n/a

### Anything in the logs that might be useful for us?

```txt
Subsystem: tty
Gerätepfad:/dev/ttyACM1
ID: /dev/serial/by-id/usb-busware.de_CUL868-if00

##########

Mounting /data as /usr/local (Home Assistant Add-On): OK
Starting watchdog...
Identifying host system: Raspberry Pi 4 Model B Rev 1.4 (oci), OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Running seedrng: OK
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.2, OK
Identifying Homematic RF-Hardware: HmRF: n/a, HmIP: n/a, OK
Updating Homematic RF-Hardware: no GPIO/USB connected RF-hardware found
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: not required
Starting rfd: no BidCos-RF hardware found
Starting HMIPServer: .......OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.73.9.20240130 (raspmatic_oci_arm64)
```

**Comment by jens-maus** [comment, marked as answer]
> Der USB Cul Stick wird von Home Assistant erkannt und als ttyACM1 angezeigt. Das Raspberrymatic Addon erkennt leider keinen Stick (Identifying Homematic RF-Hardware: HmRF: n/a, HmIP: n/a, OK). Aufgrund von [#1745](https://github.com/jens-maus/RaspberryMatic/issues/1745) bin ich davon ausgegangen, dass die Kombination funktioniert.

Falsch gedacht! Der besagte "CUL Stick" geht nur in Kombination mit dem CUxD Addon das es für CCUs/RaspberryMatic gibt. Und auch wenn man ihn mit dem CUxD Addon am laufen hat kann man damit keine normalen Homematic/homematicIP Geräte anlernen. Eine CCU/RaspberryMatic funktioniert als Homematic Geräte Zentrale **NUR** mit eQ3 eigenen Funksticks/gateways wie `HmIP-RFUSB`, `RPI-RF-MOD`. Siehe Tabelle (RTFM!) hier:

https://github.com/jens-maus/RaspberryMatic/wiki/Einleitung#voraussetzungen
